.gitignore

```diff
@@ -16,3 +16,10 @@ cypress/videos
 cypress/screenshots
 /public
 /storybook-static
+.pnp.*
+.yarn/*
```

Comment:
yet there is a file in the .yarn repository added in this PR

Reply:
Yes, look at the following lines ;-)

Reply:
I follower the yarn 2 documentation: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
right, thanks!